mmdet3d/models/detectors/single_stage_mono3d.py

```diff
@@ -8,23 +9,61 @@
 
 from mmdet3d.core import (CameraInstance3DBoxes, bbox3d2result,
                           show_multi_modality_result)
-from mmdet.models.builder import DETECTORS
-from mmdet.models.detectors.single_stage import SingleStageDetector
+from mmdet.models.detectors import BaseDetector
+from ..builder import DETECTORS, build_backbone, build_head, build_neck
 
 
 @DETECTORS.register_module()
-class SingleStageMono3DDetector(SingleStageDetector):
+class SingleStageMono3DDetector(BaseDetector):
```

Comment:
Why can we not inherit from `SingleStageDetector` any longer?

Reply:
We need to use the mmdet3d's build function instead of using mmdet build function